pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
 dependencies:
   js: ^0.6.1+1
   matcher: ^0.12.1+4
-  meta: ">=1.2.2 <1.7.0" # Workaround to avoid https://github.com/dart-lang/sdk/issues/46142
+  meta: ^1.6.0 # Workaround to avoid https://github.com/dart-lang/sdk/issues/46142
```

Comment:
#nit This comment is no longer relevant, right?
```suggestion
  meta: ^1.6.0
```

Reply:
removed the comment and the dependency validator ignore